renku/cli/dataset.py

```diff
@@ -716,13 +716,16 @@ def export_(client, id, provider, publish, tag):
     is_flag=True,
     help='Extract files before importing to dataset.'
 )
+@click.option(
+    '-y', '--yes', is_flag=True, help='Confirm unlinking of all files.'
```

Comment:
This might cause trouble with big pending refactor change, so let's get that in there or after we merge first step of refactor.